Toolchain/BuildVcpkg.bat

```diff
@@ -0,0 +1,46 @@
+@echo off
+setlocal enabledelayedexpansion
+
+REM This script builds the vcpkg dependency management system
```

Comment:
This probably works, but moving this script to be written in python for all environments might be a better idea. less maintenance.

Reply:
Please, don't bloat Ladybird environment with Python, please, please

Reply:
We already require Python. So bloated you didn't even notice. 😉

Reply:
Just because I did test latest changes? Well, now it is easier to build Chromium?

Reply:
@ADKaster Well, I saw TIFFGenerator.py already (which is buggy and not able to create missing directories), but it works for me somehow with system Python on Mac after patching. 
In case of Windows port it is an additional big dependency which requires additional maintenance, not less one.
I'm interested in Windows port and can contribute TIFFGenerator.cpp (or find another better solution) to remove that dependency. But it will not be productive if number of required Python scripts will grow.

Reply:
Using Python for build scripts is less maintenance than having both a bash script and a batch/power shell script to do the same thing. Python is a standard tool, and we will continue to use it. If there are bugs with our existing scripts, please open an issue instead of discussing it here.

Reply:
I'm trying to explain that maintaining Python itself is a nightmare on non-Linux system. While it is OK to use Python for Meta scripts which always run on Linux, it create additional maintenance cost when it is required to use in actual build.